src/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
it is a default value

Reply:
agree, remove it

Reply:
You should remove default values as there's no need to specify something that is same by default.